test/test_binary_ufuncs.py

```diff
@@ -952,13 +952,12 @@ def _wrapped_rfloordiv_scalar(a):
 
                 self.assertEqual(5 / a, scripted_rdiv_scalar(a_t))
 
-                with self.maybeWarnsRegex(UserWarning, 'floor_divide'):
-                    # Handles Issue 45199 (see comment above)
-                    if a_t.is_floating_point():
-                        with self.assertRaises(RuntimeError):
-                            scripted_rfloordiv_scalar(a_t)
-                    else:
-                        self.assertEqual(5 // a, scripted_rfloordiv_scalar(a_t))
+                # Handles Issue 45199 (see comment above)
+                if a_t.is_floating_point():
+                    with self.assertRaises(RuntimeError):
+                        scripted_rfloordiv_scalar(a_t)
+                else:
+                    self.assertEqual(5 // a, scripted_rfloordiv_scalar(a_t))
```

Comment:
`rfloordiv` actually does not emit a warning. Should it?

Reply:
That's interesting. I would have expected it to because its implemented by calling torch.floor_divide() here:

https://github.com/pytorch/pytorch/blob/c7b0005831441f223e4c95eb0ccd89fd9767789a/torch/tensor.py#L562

I think it should be throwing a warning. Maybe it's being suppressed somehow?

Reply:
Note we don't have to address that issue in this PR, but we should file an issue for it and add a comment in the code here with a link to the issue.

Reply:
File issue gh-52741